### PR TITLE
Sophia - updating currentUserFixtures to mock ROLE_DRIVER

### PR DESCRIPTION
**Pull request by SophiaWeiler**
In this PR, I am updating the currentUserFixtures to include the "driver" tag as it has in the backend and adding a new type of user "driver" that has a "ROLE_DRIVER" so that I can test and mock in the frontend while the backend is being developed.

### CODE REVIEW

**Review by strongani — COMMENTED** [review]
Do we assume that an Admin is also a Driver (or has permission of a driver)? Also, in the case I believe that for adminUser, there should be a role `"authority": "ROLE_DRIVER"` in the roles array and in the roles list at the end of the file.

**Comment by SophiaWeiler** [comment]
> Do we assume that an Admin is also a Driver (or has permission of a driver)? Also, in the case I believe that for adminUser, there should be a role `"authority": "ROLE_DRIVER"` in the roles array and in the roles list at the end of the file.

You are right, I will add the ROLE_DRIVER authority to the admin as well

File: frontend/src/fixtures/currentUserFixtures.js
Comment (strongani) [inline frontend/src/fixtures/currentUserFixtures.js:1]: On currentUserFixtures at the bottom, add ROLE_DRIVER to adminUser

**Review by strongani — CHANGES_REQUESTED** [review]

**Review by strongani — APPROVED** [review]
LGTM

**Review by pconrad — APPROVED** [review]
LGTM

**Pull request merged**